frontend/public/components/nav.jsx

```diff
@@ -357,12 +351,8 @@ export const Navigation = ({ isNavOpen, onNavSelect }) => {
             model={ClusterServiceVersionModel}
             resource={ClusterServiceVersionModel.plural}
             name="Installed Operators"
-            separatorConditions={[
-              [FLAGS.CAN_LIST_PACKAGE_MANIFEST, FLAGS.CAN_LIST_OPERATOR_GROUP, FLAGS.OPERATOR_HUB],
-              [FLAGS.CAN_LIST_PACKAGE_MANIFEST, FLAGS.CAN_LIST_OPERATOR_GROUP],
-              FLAGS.SERVICE_CATALOG,
-            ]}
           />
+          {((FLAGS.CAN_LIST_PACKAGE_MANIFEST && FLAGS.CAN_LIST_OPERATOR_GROUP) || FLAGS.SERVICE_CATALOG) && <NavItemSeparator />}
```

Comment:
We'll be able to remove this check when #1545 lands since "Operator Management" will always be visible

Reply:
Actually this isn't checking if the flag is set, only if the constant is defined.